indexing-service/src/test/java/org/apache/druid/indexing/overlord/ForkingTaskRunnerTest.java

```diff
@@ -537,6 +539,115 @@ ProcessHolder runTaskProcess(List<String> command, File logFile, TaskLocation ta
     Assert.assertTrue(forkingTaskRunner.restore().isEmpty());
   }
 
+  @Test
+  public void testTaskRun_doesnt_pass_loadLookups_Argument() throws Exception
```

Comment:
The other tests in this class don't use the underscore style of naming tests. Please adhere to the existing style in this class if possible.

Reply:
renamed